docs/Combinational/combinational.md

```diff
@@ -8,6 +8,15 @@ permalink: /docs/combinational
 # Digital Combinational Circuits
 {: .no_toc }
 
+## Table of Contents
+
+- [Half Adder](https://learn.circuitverse.org/docs/Combinational/half_adder.html)
+- [Full Adder](https://learn.circuitverse.org/docs/Combinational/full_adder.html)
+- [Half Subtractor](https://learn.circuitverse.org/docs/Combinational/half_sub.html)
+- [Full Subtractor](https://learn.circuitverse.org/docs/Combinational/full_sub.html)
+- [N-bit Parallel Adder and Subtractor](https://learn.circuitverse.org/docs/Combinational/N-Bit%20Parallel%20Adder%20&%20Subtractor.html)
+
```

Comment:
I do not think ToC's have been implemented like this on other pages.

Reply:
You were right, I have fixed the numbering now. Please find the deployment link below.